disperser/encoder/server_test.go

```diff
@@ -31,16 +31,17 @@ var logger = &cmock.Logger{}
 
 func makeTestEncoder() (*encoding.Encoder, ServerConfig) {
```

Comment:
Can we add a test where two encoders with different `SRSNumberToLoad` can encode/decode the same message?